src/Compilers/CSharp/Test/Emit3/ExtensionTypeTests.cs

```diff
@@ -54,7 +57,7 @@ private static void AssertSetStrictlyEqual(string[] expected, string[] actual)
     }
 
     // Verify things that are common for all extension types
-    private static void VerifyExtension<T>(TypeSymbol type, bool? isExplicit, SpecialType specialType = SpecialType.None) where T : TypeSymbol
+    private static void VerifyExtension<T>(TypeSymbol type, bool? isExplicit, SpecialType specialType = SpecialType.None, string fieldType = null) where T : TypeSymbol
```

Comment:
>, string fieldType = null [](http://example.com/codeflow?start=120&length=25)

Why do we need to pass this explicitly? Shouldn't the type match the `underlyingType` that we extract from `type` below?